e2e-spock-geb/Jenkinsfile

```diff
@@ -1,65 +1,17 @@
-// build params
-def odsImageTag = env.ODS_IMAGE_TAG ?: 'latest'
 def odsGitRef = env.ODS_GIT_REF ?: 'production'
-def projectId = env.PROJECT_ID
-def componentId = env.COMPONENT_ID.toLowerCase()
-def packageName = env.PACKAGE_NAME
-def gitUrlHttp = env.GIT_URL_HTTP
-def group = env.GROUP_ID
+def odsImageTag = env.ODS_IMAGE_TAG ?: 'latest'
 
-def credentialsId = "${projectId}-cd-cd-user-with-password"
-def quickstarterBasepath = "e2e-spock-geb"
-def dockerRegistry
-def gitHost
-// vars from jenkins master
-node {
-  dockerRegistry = env.DOCKER_REGISTRY
-  gitHost =  env.BITBUCKET_HOST.split(":")[0]
-}
+library("ods-jenkins-shared-library@${odsGitRef}")
+
+odsQuickstarterPipeline(
+  imageStreamTag: "cd/jenkins-slave-base:${odsImageTag}",
```

Comment:
Wrong slave image, should be `"cd/jenkins-slave-maven:${odsImageTag}"`.

Reply:
same here ...  (as below) this is purely copying files